### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by karankwatra**
**Creates Project Base and Server**
**Why:** This is necessary for the rest of the project since it creates the node package for the project and establishes a Hapi server to accept requests. 
**What:** Created a node package for the project, an entrypoint for the server in index.js, and a Hapi server for accepting requests on local port 3000.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: false
+node_js:
+  - 6.11.1
```
Comment (Lianathanoj) [inline .travis.yml]: Interesting, is there a reason why our yml here uses a different node version (v6) than what we use locally (v8)?

**Comment by Lianathanoj** [comment]
Could you rename your PR to adhere to the syntax we usually use for our commit messages? In this case, it would be `feat(base): creates project base and server`. This is important because when we squash and merge this branch into master, the PR name is what shows up on the master branch.

Also note that we want to squash and merge as opposed to just merging the request to keep the master branch cleaner. Afterwards, you'll need to rebase off master for future branches.

File: .travis.yml (hunk 1 above)
Comment (karankwatra) [reply .travis.yml]: the docs for the node package say to use v6 https://www.notion.so/lob/Node-Package-a248ae47225548ebad0f727a4fe832a2

Comment (kienan) [reply .travis.yml]: Ah, could we update those docs? We'll want to make sure that our `.node-version` always matches the version we're using in CI, so let's go for 8.11.3 in this case!

**Pull request merged**